src/logic/calculate.js

```diff
@@ -0,0 +1,133 @@
+import operate from './operate';
+
+function isNumber(item) {
+  return !!item.match(/[0-9]+/);
+}
+
+/**
+ * Given a button name and a calculator data object, return an updated
+ * calculator data object.
+ *
+ * Calculator data object contains:
+ *   total:s      the running total
+ *   next:String       the next number to be operated on with the total
+ *   operation:String  +, -, etc.
+ */
```

Comment:
- [ ] [OPTIONAL] I noticed there are a lot of comments in your code you don't really need cause your code is clear enough. I would like to recommend you remove all unnecessary comments from your code in order to make it clean comment only if needed  :smile: